vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/VerifiablePresentationFactory.kt

```diff
@@ -242,10 +237,7 @@ class VerifiablePresentationFactory(
             transactionDataHashes = if (request.transactionData?.first == PresentationRequestParameters.Flow.OID4VP) request.getTransactionDataHashes() else null,
             transactionDataHashesAlgorithm = if (request.transactionData?.first == PresentationRequestParameters.Flow.OID4VP) "sha-256" else null,
```

Comment:
I know, not a change, but while you're at it: can we get rid of the `"sha-256"` string here?